src/main/webapp/app/exercises/programming/shared/actions/programming-exercise-instructor-repo-download.component.ts

```diff
@@ -6,9 +6,7 @@ import { downloadZipFileFromResponse } from 'app/shared/util/download.util';
 import { AlertService } from 'app/core/util/alert.service';
 import { faDownload } from '@fortawesome/free-solid-svg-icons';
 
-@Component({
-    selector: 'jhi-programming-exercise-instructor-repo-download',
-    template: `
+export const template = `
```

Comment:
The same template is also used by this component's subclass [ProgrammingExerciseExampleSolutionRepoDownloadComponent](https://github.com/ls1intum/Artemis/pull/4795/files/65b0cf2f61472d9aae422786cb4182465888125c#diff-0fa3244414ba49a02a29cf4bac2ae5163789bd321d472d434deb805771c18053R7) so I wanted to reuse it instead of copying.

Reply:
I suggest do move it to a template file then and using `templateUrl` instead.

Reply:
Changed it.